packages/mui-material/src/Typography/Typography.d.ts

```diff
@@ -38,6 +38,7 @@ export interface TypographyOwnProps extends SystemProps<Theme> {
   /**
    * If `true`, the element will be a paragraph element.
    * @default false
+   * @deprecated use the `component` prop instead.
```

Comment:
We are using this format in recent deprecations:

```suggestion
   * @deprecated Use the `component` prop instead. This prop will be removed in v7. See [Migrating from deprecated APIs](/material-ui/migration/migrating-from-deprecated-apis/) for more details.
```